### PR TITLE
[SCB-1697] Add schema API in service-center-client

### PR DESCRIPTION
**Pull request by zaneChou1**
#1423  add service center schemas related API in service-center-client

### CODE REVIEW

File: clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java [hunk 1]
```diff
@@ -422,4 +425,90 @@ public void sendHeartBeats(HeartbeatsRequest heartbeatsRequest) {
...
+  /**
+   * update schema context of service
+   *
+   * @param serviceId
+   * @param schemaId
+   * @param schemaResponse
+   * @return
+   * @throws OperationException
+   */
+  public Boolean updateServiceSchemaContext(String serviceId, String schemaId, GetSchemaResponse schemaResponse) {
```
Comment (liubao68) [inline clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java]: Maybe ```boolean``` is better.

File: clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java [hunk 2]
```diff
@@ -422,4 +425,90 @@ public void sendHeartBeats(HeartbeatsRequest heartbeatsRequest) {
...
   }
+
+  /**
+   * Get schemas list of service
+   *
+   * @param serviceId
+   * @return
+   * @throws OperationException
+   */
+  public List<GetSchemaResponse> getServiceSchemasList(String serviceId) {
```
Comment (liubao68) [inline clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java]: ```GetSchemaResponse``` is used as an ```request``` in ```updateServiceSchemaContext```, it's better to change to a new name. e.g. ```SchemaInfo```

Comment (zaneChou1) [reply clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java]: Thanks for your suggestion, it has been modified.

File: clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java (hunk 1 above)
Comment (zaneChou1) [reply clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java]: Thanks for your suggestion, it has been modified.

**Pull request merged**